sechub-developertools/src/main/java/com/daimler/sechub/developertools/database/PostgreSQLTestContainer.java

```diff
@@ -0,0 +1,42 @@
+package com.daimler.sechub.developertools.database;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class PostgreSQLTestContainer extends PostgreSQLContainer<PostgreSQLTestContainer> {
+    
+    private static final String IMAGE_VERSION = "postgres:11.1";
```

Comment:
1. properties/application yaml?
This test container is NOT designed to run inside a spring boot application but as a standalone variant!  For details please look into #716 and their follow ups for jenkins and github actions.

1a: Of course this could be made configurable as well- I think the easiest way would be to add a parameter argument where callers can define the image.

2. References to postgres dockerhub image
Yes it is - I found no special docker repository settings and looking into `PostgreSQLContainer.java` from test containers I found:
`DEFAULT_IMAGE_NAME = DockerImageName.parse("postgres");`

3. Test multiple versions
When I implement the additional argument as mentioned in 1a before, this would be very simple.
We need this for #716 as well - I just added the wanted gradle parameter there.

Reply:
just added 4th parameter for image version - default is "postgres:11.1"

Reply:
done